management-system/management-system/app/components/button.tsx

```diff
@@ -0,0 +1,11 @@
+interface ButtonName {
```

Comment:
```suggestion
interface ButtonProps {
```

Because a component may have other properties as well, it’s a common practice to name the interface or type of a component with `Props` in the name